cmd/cli/plugin/secret/README.md

```diff
@@ -171,3 +171,63 @@ the "secret registry" can be used to:
      Updated registry secret 'test-secret' in namespace 'test-ns'
      Unexported registry secret 'test-secret' from all namespaces
    ```
+
+## Workflow for adding a private package repository and installation of a private package
+
+You can add a private package repository and install a private package using the following procedure:
+
+1. First, create the namespace in which the secret is getting added to:
+
+   ```sh
+   kubectl create namespace <NAMESPACE>
+   ```
+
```

Comment:
Here we are mentioning 2 & 3 separately(only one way). 
We can start by saying there are 2 ways of adding/exporting a secret after Step 1:
1. 2& 3 separately ( where 2 adds secret and 3 updates secret by exporting to a diff namespace)
or
specify a single one line command for doing adding/exporting together